composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.0",
         "fsi/datasource" : "~1.2",
-        "ruflin/elastica": "~1.2",
+        "ruflin/elastica": "~2.3",
```

Comment:
I never said it isn't a BC break. This driver is proved to work in several projects so there is a good chance to tag `1.0.0` now